include/deal.II/base/exceptions.h

```diff
@@ -1191,9 +1216,13 @@ namespace StandardExceptions
     ExcNeedsAssimp,
     "You are attempting to use functionality that is only available "
     "if deal.II was configured to use Assimp, but cmake did not "
-    "find a valid Assimp library.");
-
-  /** @} */
```

Comment:
This line was misplaced. It should have come *after* all of the exceptions are declared. So I moved it down to the end of the list of exceptions.